### DISCUSSION TITLE
Certificate Verification fails, cannot open webpage?

### DISCUSSION THREAD

**Discussion by natejw** [ente-io/ente#3590] [Q&A]
Certificate Verification fails. 
![image](https://github.com/user-attachments/assets/eae8b7eb-1218-4e5c-b14a-47e16eeab46d)
so I click update verification and it opens a html window but windows complains that it can't find the file.
![image](https://github.com/user-attachments/assets/8cc57a09-5626-41f0-ba5d-8f0994eec67e)
Also nowhere to be found in explorer.
![image](https://github.com/user-attachments/assets/fc147c04-5fd0-4a08-a725-e8acaa461e41)

All help would be appreciated!

**Comment by natejw** [comment]
Just realised I captalised the explorer search, same issue still occurs though.

**Comment by ua741** [comment]
Please refer to following doc: https://help.ente.io/auth/troubleshooting/windows-login
